File: atividade-4/014_elemento/014_elemento.js
//Pegar elemento do vetor

const n = -4;
const list = [0, 1, 2, 3, 4, 5];

const elemento = (n, xs) => {
  const pos = n < 0 ? n + xs.length : n;

  return xs[pos];
};

console.log(elemento(n, list));

File: atividade-4/014_elemento/014_elemento.hs
module Main where

-- Pegar elemento do vetor

elemento :: Int -> [Int] -> Int
elemento n xs = xs!!pos
    where
      tam = length xs
      pos
        | n < 0 = n + tam
        | otherwise = n

main = do
  a <- readLn :: IO Int
  b <- readLn :: IO [Int]
  print $ elemento a b
